contracts-periphery/.env.example

```diff
@@ -2,6 +2,6 @@ MAINNET_RPC_URL=yourMainnetRpcUrlHere
 POLYGON_RPC_URL=
 OPTIMISM_RPC_URL=
 ARBITRUM_ONE_RPC_URL=
+GNOSIS_CHAIN_RPC_URL=
```

Comment:
CI is failing since the `GNOSIS_CHAIN_RPC_URL` var is missing. You'll need to set it in the github secrets and the CI yaml file

Reply:
resolved in amended commit